### DISCUSSION TITLE
operation not permitted, rmdir

### DISCUSSION THREAD

**Discussion by gmmedia** [GetPublii/Publii#1579] [Bug Report]
### Operating system

Win 11

### Publii version

0.45.2 (build 16609)

### Issue type

Application

### Bug description

Hello!
I have changed the files location folder.
After that, the preview was not possible anymore. It stops at 85% with the following error:
```
Rendering process failed
An error occurred during rendering process:

EPERM: operation not permitted, rmdir 'C:\Stuff\Publii\Preview\media\authors'
```

Changing it back didn't solve the issue.
I think importing the site from a backup is the only way to fix it.

**Comment by candidexmedia** [comment]
How did you change the location, from the App Settings?

**Reply by gmmedia** [reply]
Yes, in the App settings.

**Comment by gmmedia** [comment]
Seems like another user has the same problem: https://github.com/GetPublii/Publii/discussions/1535

**Reply by slymeasy** [reply]
Were you ever able to solve this problem? I'm having the same issue. It has something to do with electron...

```
Rendering process failed
An error occurred during rendering process:

EPERM: operation not permitted, rmdir 'D:\Websites\Publii Sites\xxxx\preview\media\authors'

Error: EPERM: operation not permitted, rmdir 'D:\Websites\Publii Sites\xxxx\preview\media\authors'
```

**Reply by dziudek** [reply]
@slymeasy @gmmedia - after analysis I think that it must be related to the `delete-empty` module which is used to remove empty catalogs from the media directory. Probably it has some issues on Windows - I will try to replace it in v.0.46.2
